Library/Formula/nspr.rb

```diff
@@ -32,12 +34,12 @@ def install
         --enable-strip
         --enable-pthreads
         --enable-ipv6
-        --enable-macos-target=#{MacOS.version}
       ]
-      args << "--enable-64bit" if MacOS.prefer_64_bit?
+      args << "--enable-macos-target=#{MacOS.version}" if OS.mac?
+      args << "--enable-64bit" if MacOS.prefer_64_bit? || OS.linux?
       system "./configure", *args
       # Remove the broken (for anyone but Firefox) install_name
-      inreplace "config/autoconf.mk", "-install_name @executable_path/$@ ", "-install_name #{lib}/$@ "
+      inreplace "config/autoconf.mk", "-install_name @executable_path/$@ ", "-install_name #{lib}/$@ " if OS.mac?
```

Comment:
This is the line that might trigger the couchdb/spidermonkey bug I mentioned in the PR description. I've just disabled it entirely now, since I couldn't find any instance of "-install_name" in `autoconf.mk` and the line fails otherwise.

We might have to fix this later when we fix the spidermonkey build, but IMO it's fine for now.

Thoughts?